.env.example

```
@@ -0,0 +1,7 @@
+DB_URL=localhost:5439/delivery_db
+DB_USERNAME=postgres
+DB_PASSWORD=postgres
+EUREKA_SERVER_URL=http://localhost:18761/eureka
+BOOTSTRAP_SERVERS=<GCP Kafka broker public IPs (콤마 구분)>
+KAFKA_BOOTSTRAP_SERVERS=<위와 동일>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)